crates/sqlbuiltins/src/functions/mod.rs

```diff
@@ -349,6 +362,16 @@ impl FunctionRegistry {
         }
         None
     }
+
+    pub fn register_udf(&mut self, udf: Arc<dyn BuiltinScalarUDF>) {
```

Comment:
this should probably error/panic if you attempt to register a function that has already been registered

Reply:
the catalog mutator already will error if it already exists

`catalog_mutator.mutate(catalog_version, mutations).await?;`